README.md

```diff
@@ -75,11 +75,11 @@ print(training_df.head())
 # model = ml.fit(training_df)
 ```
 ```commandline
-      event_timestamp  driver_id  driver_hourly_stats__conv_rate  driver_hourly_stats__acc_rate
-  2021-04-12 08:12:10       1002                        0.497279                       0.357702
```

Comment:
how come these values are changed? were they out of date and different from what you get when you run these commands with the sample data?

Reply:
I reran the code above to get the output in the new format (non-prefixed feature names) and these were the values I got. When I rerun the readme instructions from scratch in a new repository, I obtain a new set of values so I believe a different set is generated each time you create a feature repository using `feast init` (?).

I think the example is moreso to display the expected format of the output, although I'm not sure why the original example was missing a `avg_daily_trips` column in its output.

Reply:
cool got it